config/blobs.yml

```diff
@@ -1,16 +1,12 @@
-go/go1.21.6.linux-amd64.tar.gz:
```

Comment:
is there any specific reason to remove golang blob?

Reply:
Yes, go is being bumped as bosh vendor. so removed it form here .

Reply:
okay